src/Compilers/CSharp/Test/Symbol/Symbols/Source/EnumTests.cs

```diff
@@ -349,10 +349,17 @@ static void Main(string[] args)
 ";
             VerifyEnumsValue(source, "c1.Suits", 1, 2, 4, 2, 2);
 
+#if NETCOREAPP
+            CompileAndVerify(source, expectedOutput: @"
```

Comment:
Instead of testing that `Enum.ToObject` chooses a specific enum member for the value `2`, could we test that `Console.WriteLine(S == Suits.ValueB);` outputs `true`? Could even additionally test that `Console.WriteLine(S == Suits.ValueE);` also outputs `true`.

Reply:
Same suggestion for the VB version of this test.